### DISCUSSION TITLE
mpv: deleting the video works but then a browser attempts to open it

### DISCUSSION THREAD

**Discussion by rieje** [netblue30/firejail#4788] [Q&A]
I have a [simple script](https://0x0.st/-C6A.txt) that trashes a file and creates an empty placeholder file of the same name for archiving (see the usage function if curious). I bound this script to a key for mpv:

    # Just file to trash can, do not archive
    DEL run "trash-archive" -n "${path}"; quit

So watching the video, I press DEL to delete the video and quit mpv. This worked for quite a while but recently, it had a strange behavior where it would delete the video but a browser (Firefox/Chromium) would open and attempt to open the now non-existent behavior. I have no idea what's causing this but I assume it's Firejail-related because **I remove /usr/local/bin/mpv to disable Firejail and the strange behavior is gone**. It also looks like it may be file opener related?

My [~/.config/firejail/mpv.local](https://0x0.st/-C6a.txt).

Any ideas on what may be the problem or how to diagnose before submitting bug report is appreciated.

**Comment by ghost** [comment]
> Any ideas on what may be the problem or how to diagnose before submitting bug report is appreciated.

Personally I would start debugging by `joining` the mpv sandbox. Add `name mpv` to your mpv.local and run `firejail --join=mpv` when mpv is playing the file (or pause it to focus on the debugging). Once you're in, **manually run the command you bound to a key and watch the script flow**.

Not sure if it has anything to do with the unexpected opening of Firefox/Chromium, but I notice that your script defines `delete_dir="$HOME/deleted-files-archive"` and your mpv.local has:
```
# whitelist the parent dir of $delete_dir in ~/bin/trash-archive
whitelist ${HOME}/misc
```
This seems odd to me at first glance. AFAICT `${HOME}/deleted-files-archive` is not whitelisted anywhere is it?
